src/store/index.js

```diff
@@ -140,6 +150,44 @@ const store = {
     userPasswordExists: sessionStorage.getItem('userPassword') !== null
   },
   actions: {
+    createStubMessage (state, payload) {
+      const timestamp = utils.epochTime()
+      // Build chat message
+      let message = payload.message
+
+      let contacts = Object.entries(state.getters.getContacts.list)
+      let ADMAddress
+      contacts.forEach((contact) => {
+        const ethAddress = contact[1].ETH
```

Comment:
How is this supposed to work with non-ETH coins (e.g. DOGE)? Make this code more versatile.